File: haskell/exercises1/ex6/ex6.hs
-- Programa que receba um valor e verifique se ele está em uma lista

inList _ [] = False
inList a (x:xs) = if x == a 
    then True 
    else inList a xs

main = do
    let list = [1,2,3,4,5]
    let elem = 2
    let res = inList elem list
    print res

File: prolog/exercises1/ex6.pl
% Programa que receba uma lista e um item, retorna uma nova lista com o item removido uma vez 

remove(_, [], []).
remove(X, [X|T], T).

remove(X, [H|T], R) :- 
    remove(X, T, RR),
    R = [H|RR].
